.github/workflows/CI.yml

```diff
@@ -25,33 +25,35 @@ jobs:
       matrix:
         gap-branch:
           - master
+          - stable-4.12
           - stable-4.11
           - stable-4.10
           - stable-4.9
```

Comment:
```suggestion
```
Dropping tests on 4.9 because the `automata` version there is too old (we could install a newer version but it doesn't seem worth the bother)